resources/js/Pages/Users/Index.vue

```diff
@@ -228,7 +271,17 @@ const props = defineProps({
   filters: Object,
 })
 
+const toast = useToast()
 const search = ref(props.filters.search)
+const selectedUsers = ref([])
+const indeterminate = computed(() => selectedUsers.value.length > 0 && selectedUsers.value.length < props.users.data.length)
+
+function copyEmails(){
+  const emails = selectedUsers.value.join(', ')
```

Comment:
Could we build something in this form: `"John Doe" <john@example.com>, "John Doe2" <john2@example.com>?`